packages/snaps-sdk/src/index.ts

```diff
@@ -1 +1,2 @@
+export type { EnumToUnion } from './internals';
```

Comment:
Maybe leave a comment about the goal here, we can export some of the internals as needed, but ideally we dont